site/pages/cookie-details.tsx

```diff
@@ -0,0 +1,199 @@
+import Link from "next/link";
+import { ReactElement } from "react";
+import { TwoThirdsLayout } from "../components/layout/Layout";
+
+const title = "Cookie details - Create Transport Disruption Data Service";
+const description = "Cookie details page for the Create Transport Disruption Data Service";
+
+const Contact = (): ReactElement => {
+    return (
```

Comment:
fdbt is found in this page a few times also

Reply:
removed